test/dispatchers/mouseDispatcherTests.ts

```diff
@@ -178,117 +38,219 @@ describe("Dispatchers", () => {
         y: targetY
       };
 
-      let md = Plottable.Dispatchers.Mouse.getDispatcher(<SVGElement> target.node());
+      let svg: d3.Selection<void>;
+      let mouseDispatcher: Plottable.Dispatchers.Mouse;
+
+      beforeEach(() => {
+        let SVG_WIDTH = 400;
+        let SVG_HEIGHT = 400;
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        // HACKHACK: PhantomJS can't measure SVGs unless they have something in them occupying space
+        svg.append("rect").attr("width", SVG_WIDTH).attr("height", SVG_HEIGHT);
 
-      let callbackWasCalled = false;
-      let callback = (p: Plottable.Point, e: MouseEvent) => {
-        callbackWasCalled = true;
-        TestMethods.assertPointsClose(p, expectedPoint, 0.5, "mouse position is correct");
-        assert.isNotNull(e, "mouse event was passed to the callback");
-      };
-
-      md.onMouseDown(callback);
+        mouseDispatcher = Plottable.Dispatchers.Mouse.getDispatcher(<SVGElement> svg.node());
+      });
 
-      TestMethods.triggerFakeMouseEvent("mousedown", target, targetX, targetY);
-      assert.isTrue(callbackWasCalled, "callback was called on mousedown");
+      it("calls the mouseDown callback", () => {
+        let callbackWasCalled = false;
+        let callback = (point: Plottable.Point, event: MouseEvent) => {
+          callbackWasCalled = true;
+          TestMethods.assertPointsClose(point, expectedPoint, 0.5, "mouse position is correct");
+          assert.isNotNull(event, "mouse event was passed to the callback");
```

Comment:
we probably didn't but I think it's good for completeness
personally I would prefer if you can test if it's the same `event` that you created via `TestMethods.triggerFakeMouseEvent`but that could be messy and requires some rewrite in `TestMethods`
so I am okay with checking if its type as bare minimum